projects/plugins/jetpack/wp-cli-templates/block-block-json.mustache

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 1,
```

Comment:
@monsieur-z What's your take on this?

Reply:
It makes sense to nudge people to use the latest version when possible. In this case (using 2 or 3), we'd probably need to edit the `save` and `edit` functions in the templates to include `useBlockProps();` (as in the link Brad copied) and make it visible it's required.

As for which version to use, I can't say I fully understand what [using version 3](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-api-versions/) involves. The iframing might be an issue for some, but it seems it wouldn't occur unless all blocks are registered with 3 anyway.

Reply:
What I've managed to gather is that we have to make sure the scripts will be enqueued inside the iframe. Which it seems may involve somehow using the 'enqueue_block_assets' hook rather than 'enqueue_block_editor_assets', or else doing the enqueuing via block.json.

https://developer.wordpress.org/block-editor/how-to-guides/enqueueing-assets-in-the-editor/#editor-content-scripts-and-styles (and [the following section](https://developer.wordpress.org/block-editor/how-to-guides/enqueueing-assets-in-the-editor/#block-scripts-and-styles)) might be relevant.